app/services/open_access_service.rb

```diff
@@ -11,7 +11,7 @@ def unrestricted_use_files?
     # Scholarly articles
     when Article, ConferenceItem, GenericWork
       return true if peer_reviewed? &&
-                     work.record_level_file_version_declaration &&
+                     work.record_level_file_version_declaration == '1' &&
```

Comment:
Do we have a constant for this magic string?

Reply:
I don't understand your question.  `record_level_file_version_declaration` is defined on the input form as boolean, and is stored as either '0' or '1' so how is this a `magic string`?